File: 217-contains-duplicates/sol.cpp
#include <iostream>
#include <set>
#include <vector>

class Solution
{
public:
    bool containsDuplicate(vector<int> &nums)
    {
        set<int> ans;
        for (int i : nums)
        {
            ans.insert(i);
        }
        if (ans.size() == nums.size())
            return false;
        return true;
    }
}

int
main()
{
    return 0;
}

File: 9-Palindrome-Number/sol.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x<0:
            return False
        y = str(x)
        ans=""
        for i in range(len(y)-1,-1,-1):
            ans+=y[i]
        if ans==y:
            return True
        return False